jobs_test.go

```diff
@@ -33,33 +34,45 @@ var (
 	cfg *config.Config
 )
 
-func _inittests(t *testing.T, memoize bool) *Job {
+func _initconfig(t *testing.T) {
+	var err error
+	cfg, err = configurate.Init("test/test_config.yaml")
+	if err != nil {
+		t.Fatal(err)
```

Comment:
I made this error fatal so that it would tests where the configuration could not be loaded to exit immediately instead of continuing and (probably) displaying another less helpful error. I didn't see any potential negative side effects of making the error fatal, but I'd like another set of eyes on this to be sure.